core/src/kura.rs

```diff
@@ -50,13 +51,15 @@ pub struct GetBlockHash {
 /// Provides all necessary methods to read and write data, hides implementation details.
 #[derive(Debug)]
 pub struct KuraWithIO<W: WorldTrait, IO> {
+    // TODO: Kura doesn't have different initialisation modes!!!
+    #[allow(dead_code)]
     mode: Mode,
     block_store: BlockStore<IO>,
     merkle_tree: MerkleTree<VersionedCommittedBlock>,
     wsv: Arc<WorldStateView<W>>,
     broker: Broker,
     mailbox: usize,
-    io: IO,
+    io: PhantomData<IO>,
```

Comment:
what's the reason for having this `PhantomData`?

Reply:
Sorry, force of habit.